### PR TITLE
release 0.13.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file.
...
+
+**Fixed bugs:**
+
+- \[warn\] the "ssl" directive is deprecated, use the "listen ... ssl" directive instead [\#1224](https://github.com/voxpupuli/puppet-nginx/issues/1224)
+- dont deploy "ssl on" on nginx 1.15 or newer [\#1225](https://github.com/voxpupuli/puppet-nginx/pull/1225) ([bastelfreak](https://github.com/bastelfreak))
+
+**Merged pull requests:**
+
+- README: Remove old email address [\#1223](https://github.com/voxpupuli/puppet-nginx/pull/1223) ([3flex](https://github.com/3flex))
+- Update location.pp [\#1220](https://github.com/voxpupuli/puppet-nginx/pull/1220) ([swenske](https://github.com/swenske))
```
Comment (ekohl) [inline CHANGELOG.md]: I've modified the title to be a bit more descriptive

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +44,7 @@ These should not affect the functionality of the module.
 
 **Merged pull requests:**
 
+- release 0.12.0 [\#1215](https://github.com/voxpupuli/puppet-nginx/pull/1215) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (ekohl) [inline CHANGELOG.md]: Marked as skip-changelog now

File: CHANGELOG.md [hunk 3]
```diff
@@ -1264,5 +1285,7 @@ These should not affect the functionality of the module.
 
 
 
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
+
 
-\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
\ No newline at end of file
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```
Comment (wyardley) [inline CHANGELOG.md]: is it intentional that this is in here twice?

Comment (bastelfreak) [reply CHANGELOG.md]: fixed

**Comment by bastelfreak** [comment]
okay, msync fails because puppet-gitlab-ci-runner is an empty repo? https://travis-ci.org/voxpupuli/modulesync_config/builds/401538315#L2173

**Comment by bastelfreak** [comment]
Addresses all points from @ekohl and @wyardley

**Comment by wyardley** [comment]
FWIW, I see the double "automatically generated" message still.

**Comment by bastelfreak** [comment]
mhm strange, I fixed that locally before releasing

**Pull request merged**